javaSE/src/main/java/com/smartdevicelink/managers/lifecycle/LifecycleManager.java

```diff
@@ -434,6 +442,19 @@ public void run() {
 
     };
 
+    private void processLanguageChange(){
```

Comment:
for some reason I am getting this error when I change the language on the sdl_hmi:
```
ERROR: SdlSession - on protocol error - Attempting to start without setting a transport manager.	at java.base/java.lang.Object.wait(Native Method)
	at java.base/java.lang.Thread.join(Thread.java:1361)
	at org.java_websocket.server.WebSocketServer.stop(WebSocketServer.java:258)
	at com.smartdevicelink.transport.WebSocketServer.stop(WebSocketServer.java:83)
	at com.smartdevicelink.transport.TransportManager.close(TransportManager.java:85)
	at com.smartdevicelink.protocol.SdlProtocolBase.endSession(SdlProtocolBase.java:532)
	at com.smartdevicelink.SdlConnection.SdlSession.close(SdlSession.java:107)
	at com.smartdevicelink.managers.lifecycle.LifecycleManager.processLanguageChange(LifecycleManager.java:448)
	at com.smartdevicelink.managers.lifecycle.LifecycleManager.access$900(LifecycleManager.java:71)
	at com.smartdevicelink.managers.lifecycle.LifecycleManager$3.onReceived(LifecycleManager.java:430)
	at com.smartdevicelink.managers.lifecycle.LifecycleManager.onRPCReceived(LifecycleManager.java:477)
	at com.smartdevicelink.managers.lifecycle.LifecycleManager.access$1100(LifecycleManager.java:71)
	at com.smartdevicelink.managers.lifecycle.LifecycleManager$4.onProtocolMessageReceived(LifecycleManager.java:833)
	at com.smartdevicelink.SdlConnection.SdlSession.onProtocolMessageReceived(SdlSession.java:250)
	at com.smartdevicelink.protocol.SdlProtocolBase$MessageFrameAssembler.handleSingleFrameMessageFrame(SdlProtocolBase.java:1452)
	at com.smartdevicelink.protocol.SdlProtocolBase$MessageFrameAssembler.handleFrame(SdlProtocolBase.java:1328)
	at com.smartdevicelink.protocol.SdlProtocolBase.handlePacketReceived(SdlProtocolBase.java:667)
	at com.smartdevicelink.protocol.SdlProtocolBase$2.onPacketReceived(SdlProtocolBase.java:1130)
	at com.smartdevicelink.transport.TransportManager$SingleTransportCallbackImpl.onPacketReceived(TransportManager.java:210)
	at com.smartdevicelink.transport.WebSocketServer.onMessage(WebSocketServer.java:169)
	at org.java_websocket.server.WebSocketServer.onWebsocketMessage(WebSocketServer.java:580)
	at org.java_websocket.drafts.Draft_6455.processFrame(Draft_6455.java:715)
	at org.java_websocket.WebSocketImpl.decodeFrames(WebSocketImpl.java:367)
	at org.java_websocket.WebSocketImpl.decode(WebSocketImpl.java:212)
	at org.java_websocket.server.WebSocketServer$WebSocketWorker.run(WebSocketServer.java:925)
```

Reply:
that's part of an InterruptionException that is caught when the web socket thread is stopped. It will re-launch after